internal/api/cluster.go

```diff
@@ -262,9 +288,9 @@ func handleGetCluster(c *Context, w http.ResponseWriter, r *http.Request) {
 	outputJSON(c, w, cluster)
 }
 
-// handleUpdateCluster responds to PUT /api/cluster/{cluster}, updating a cluster's
+// handleUpdateClusterConfiguration responds to PUT /api/cluster/{cluster}, updating a cluster's
 // configuration.
-func handleUpdateCluster(c *Context, w http.ResponseWriter, r *http.Request) {
+func handleUpdateClusterConfiguration(c *Context, w http.ResponseWriter, r *http.Request) {
```

Comment:
Is this change OK? I think it makes the meaning of the method clearer.

Reply:
Fine with me!